pybamm/input/parameters/lithium_ion/negative_electrodes/graphite_ORegan2021/graphite_LGM50_electrolyte_exchange_current_density_ORegan2021.py

```diff
@@ -1,7 +1,9 @@
 from pybamm import exp, constants, Parameter
 
 
-def graphite_LGM50_electrolyte_exchange_current_density_ORegan2021(c_e, c_s_surf, T):
+def graphite_LGM50_electrolyte_exchange_current_density_ORegan2021(
+    c_e, c_s_surf, T, c_s_max
```

Comment:
Very picky comment (and maybe quite annoying to implement). Do we want to flip `T` and `c_s_max`? This way we have electrochemical variables first, and temperature last.